package.json

```diff
@@ -29,7 +29,7 @@
     "eslint": "^6.8.0",
     "locate-character": "^2.0.5",
     "mocha": "^7.1.1",
-    "rollup": "^0.67.4",
+    "rollup": "^1.32.1",
```

Comment:
If we're bumping the Rollup version, should we bump it to 2.x?

Reply:
Well it was shbmitted 8 months ago and that was the latest version at the time, I can spend some time to upgrade to latest.

Reply:
I tested and there's no code change required to bump it to 2.x. But maybe we should just leave it at 1.x since some users may not have upgraded yet